packages/template-retail-react-app/jest-setup.js

```diff
@@ -50,6 +50,12 @@ export const setupMockServer = () => {
         rest.get('*/customers/:customerId/baskets', (req, res, ctx) =>
             res(ctx.delay(0), ctx.status(200), ctx.json(mockCustomerBaskets))
         ),
+        rest.get('*/shopper-context/:usid', (req, res, ctx) =>
+            res(ctx.delay(0), ctx.status(200), ctx.json({}))
+        ),
+        rest.get('*/shopper-experience/*', (req, res, ctx) =>
+            res(ctx.delay(0), ctx.status(200), ctx.json({}))
+        ),
```

Comment:
This will fix this error
![image](https://github.com/user-attachments/assets/5079d24a-7998-402b-be5c-9238f1f7814f)